_guide/_pages/interviews/code-review.md

```diff
@@ -29,16 +29,17 @@ The focus of the interview should be on their code, how it works, why they made
 
 - Whenever possible, ask these questions **exactly as they’re worded** to try to get consistency between multiple candidates.
 
-- For more information on interviewing in general, check out the [interviewing overview guide](..).
 
 ## Before the interview
 
-Before the interview, prepare by reviewing the code, trying to run it (and the tests), and thinking about questions you'll want to ask the candidate during the interview (see below).
+Before the interview, prepare by reviewing the code, trying to run it (and the tests), and review the list of what to look for and the questions.
 
 **As you review, remember that candidate only spent about three hours on this code.** It doesn't need to be perfect. It should work, and solve the problem assigned, but minor issues, rough edges, and less-than-perfect parts are totally fine. A good way to think about this: you're looking for code that's about the same level as you might expect on an initial pull request. It might not be perfect yet, but that's what code review is for.
 
 For the list of things to look for during code review, see [what to look for during code review](https://docs.google.com/document/d/1rP7_yYr_IbfDi-Tj-1troJG2rTDcfVRnI_iH7Ngrzds/edit#heading=h.6l09j457zrt).
 
+<a class="usa-button usa-button-primary" href="https://docs.google.com/document/d/1rP7_yYr_IbfDi-Tj-1troJG2rTDcfVRnI_iH7Ngrzds/edi/copy">Make a copy for your notes</a>.
```

Comment:
Looks like there’s a typo at the end of this URL. 

![keyboard cat](https://media.giphy.com/media/Hcw7rjsIsHcmk/giphy.gif)

Reply:
thanks for catching!